app/views/organisations/index.html.erb

```diff
@@ -1,33 +1,35 @@
-<h1 class="govuk-heading-l">Signed Up Organisations: <%= @all_organisations.count %></h1>
+<h1 class="govuk-heading-l">All organisations</h1>
+<h3 class="govuk-heading-m">
+  <%= pluralize(@all_organisations.count, 'organisation') %> are signed up to GovWifi
+</h3>
+
 <table class="govuk-table">
   <thead class="govuk-table__head">
     <tr class="govuk-table__row">
       <th class="govuk-table__header">Name</th>
       <th class="govuk-table__header">Created on</th>
-      <th class="govuk-table__header">Service Email</th>
-      <th class="govuk-table__header">MoU</th>
+      <th class="govuk-table__header govuk-table__header">MOU</th>
+      <th class="govuk-table__header govuk-table__header--numeric">Locations</th>
+      <th class="govuk-table__header govuk-table__header--numeric">IPs</th>
     </tr>
   </thead>
   <tbody class="govuk-table__body">
     <% @all_organisations.each do | organisation| %>
       <tr class="govuk-table__row">
-        <td class="govuk-table__cell" scope="row"><%=  organisation.name %></td>
-        <td class="govuk-table__cell" scope="row"><%=  organisation.created_at.strftime('%e %b %Y') %></td>
-        <td class="govuk-table__cell" scope="row"><%=  truncate(organisation.service_email) %></td>
-        <td class="govuk-table__cell" scope="row">
-          <div>
-            <% if organisation.signed_mou.attached? %>
-              <%= link_to 'Download', rails_blob_path(organisation.signed_mou, disposition: "attachment") %>
-            <% else %>
-              No file uploaded yet
-            <% end %>
-          </div>
-          <div>
-            <%= form_with url: admins_mou_index_path(id: organisation.id), method: :post do |form| %>
-              <%= form.file_field(:signed_mou) %>
-              <%= form.submit('Upload') %>
-            <% end %>
-          </div>
+        <td class="govuk-table__cell name-col" scope="row">
+          <%= organisation.name %>
+        </td>
+        <td class="govuk-table__cell created-date-col" scope="row">
+          <%= organisation.created_at.strftime('%e %b %Y') %>
+        </td>
+        <td class="govuk-table__cell govuk-table__cell mou-col" scope="row">
+          <%= organisation.signed_mou.present? ? 'uploaded' : 'none' %>
```

Comment:
I don't think `.present?` works how I expect on an `activestorage` method.

Reply:
Just FYI, the method is `.attached?`